python/exercise1/vowel_counter.py

```diff
@@ -2,17 +2,19 @@
 
 def num_vowels(text):
     """Return the number of vowels in string."""
-    vowels = "aeiou"
+    vowels = "aeiouy"
     num = 0
     for v in vowels:
         num += text.lower().count(v)        
     return num
 
 def num_consonants(text):
-    vowels = "aeiou"
+    vowels = "aeiouy"
+    num = 0
     for letter in text:
         if letter not in vowels:
-            print("consonant", letter)
+           num += text.lower().count(letter)
```

Comment:
This will count all instances of the letter each time that letter is counted, which will lead to too high of a count. Suggest adding unit tests to test that we're getting the expected outputs for some simple examples. These unit tests will help us check that the bug is fixed and also be useful in future to check that future changes don't break the code.